### ISSUE TITLE
TypeError on simple test

### ISSUE THREAD

**Issue by bogatovam** [audeering/opensmile-python#24]
Hi

I tried the next code and got TypeError:
```
smile = opensmile.Smile(
        feature_set=opensmile.FeatureSet.ComParE_2016,
        feature_level=opensmile.FeatureLevel.Functionals,
    )
    y = smile.process_file("../test.wav")
    print(y)
```
Error:
```
Traceback (most recent call last):
  File "main.py", line 55, in <module>
    y = smile.process_file(file="../test.wav", channel=0)
  File "\venv\lib\site-packages\opensmile\core\audinterface.py", line 349, in process_file
    file, start=start, end=end, channel=channel,
  File "\venv\lib\site-packages\opensmile\core\audinterface.py", line 103, in process_file
    file, start=start, end=end, channel=channel,
  File "\venv\lib\site-packages\opensmile\core\audinterface.py", line 70, in _process_file
    file=file,
  File "\venv\lib\site-packages\opensmile\core\audinterface.py", line 201, in _process_signal
    **self.process_func_kwargs,
  File "\venv\lib\site-packages\opensmile\core\smile.py", line 218, in _extract
    config.EXTERNAL_SOURCE_COMPONENT, bytes(x)
TypeError: only integer scalar arrays can be converted to a scalar index
```

How can i fix this?

**Comment by frankenjoe** [comment]
Could you please try f the following works for you:

```python
import audiofile

x, sr = audiofile.read('../test.wav')
y = smile.process_signal(x, sr)
print(y)
```

**Comment by bogatovam** [comment]
No it dont help. But I fixed this be switching to python 3.8 from 3.6, thank you
